docs/run-platform-server/monitoring.mdx

```diff
@@ -7,11 +7,54 @@ import { CodeExample } from "@site/src/components/CodeExample";
+
+In the case of Horizon, metrics are published on the Admin http port which is bound to the host machine's loop back network interface(127.0.0.1), so external monitoring systems or services cannot reach the port directly. To expose the metrics securely, we recommend following the exporter pattern, which is common metrics scraping strategy.
+
+One real world example for exporting from a bare metal horizon installation (horizon has been installed directly onto an operating system), use [FluentBit and the Prometheus Exporter](https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-exporter) on the same host machine as Horizon is running. FluentBit will perform a simple port forwarding pipeline on the host machine. Configure the input to be Horizon's `localhost:<ADMIN_PORT>/metrics` and the output to be the `host` and `port` representing the target network interface and port on the host machine, and then configure your monitoring infrastructure to scrape that host address.
+
+In container orchestrated environents such as kubernetes, you can use the same exporter strategy. We assume you already have a metrics infrastructure deployment like Prometheus and Grafana setup on the cluster via the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator), and will just need to configure that infrastructure to scrape the horizon pod based on the ADMIN_PORT.
+
+### Data model
+
+There are numerous application metrics keys emitted by horizon at runtime, encoded in four types of exposition formats: `counter`, `gauge`, `histogram`, `summary`. Each key is further qualified with labels for more granularity. To summarize, we can highlight the groupings of metrics keys by common function denoted in the prefix of their name:
+
+- `go_`: golang specfici runtime performance
+- `horizon_txsub_`: attributes of horizon transaction submission sub system if enabled.
+- `horizon_stellar_core_`: runtime attributes of stellar network reported by the captive core or a remote stellar core which horizon is configured.
+- `horizon_order_book_`: runtime attributes of the in memory order book maintained by horizon of the current stellar network
+- `horizon_log_`: counters of how many log messages printed at each severity level
+- `horizon_ingest_`: performance measurements and stateful aspects of horizon's internal ingestion sub system
+- `horizon_http_`: statistics and measurements of horizon's HTTP API service, all aspects of request/response load and timings.
+- `horizon_history_`: statistics on horizon ingested historical ledgers
+- `horizon_db_`: measurements on database performance, query times per endpoints, pooling stats
+- `process_`: generic host machine compute measurements
```

Comment:
Should I be seeing metrics with "process" in the name?  I can't find it when I look at the metrics endpoint of my local Horizon.

Reply:
I think so, I saw the following in response:

```
# HELP process_cpu_seconds_total Total user and system CPU time spent in seconds.
# TYPE process_cpu_seconds_total counter
process_cpu_seconds_total 5206.53
# HELP process_max_fds Maximum number of open file descriptors.
# TYPE process_max_fds gauge
process_max_fds 8192
# HELP process_open_fds Number of open file descriptors.
# TYPE process_open_fds gauge
process_open_fds 49
# HELP process_resident_memory_bytes Resident memory size in bytes.
# TYPE process_resident_memory_bytes gauge
process_resident_memory_bytes 1.196032e+08
# HELP process_start_time_seconds Start time of the process since unix epoch in seconds.
# TYPE process_start_time_seconds gauge
process_start_time_seconds 1.69142892499e+09
# HELP process_virtual_memory_bytes Virtual memory size in bytes.
# TYPE process_virtual_memory_bytes gauge
process_virtual_memory_bytes 1.110106112e+09
# HELP process_virtual_memory_max_bytes Maximum amount of virtual memory available in bytes.
# TYPE process_virtual_memory_max_bytes gauge
process_virtual_memory_max_bytes -1
```